plugins/scanners/scancode/src/test/kotlin/ScanCodeResultParserTest.kt

```diff
@@ -80,9 +80,11 @@ class ScanCodeResultParserTest : FreeSpec({
 
                 val summary = parseResult(resultFile).toScanSummary()
 
-                summary.licenseFindings.find {
-                    it.location == TextLocation("README.md", 100) && it.score == 100.0f
-                }?.license.toString() shouldBe "GPL-2.0-only WITH GCC-exception-2.0"
+                with(summary.licenseFindings) {
+                    shouldHaveSize(18)
```

Comment:
commit-msg: Please provide a rationale. E.g. during scancode upgrade this test would fail because the findings changed to 19, I would not know what the 18 is about.

Reply:
Note that we're parsing a committed test asset here, so the number of findings would not change unless we also change the committed result file.